packages/core/saved-objects/core-saved-objects-server/src/mapping_definition.ts

```diff
@@ -64,19 +65,20 @@ export interface SavedObjectsMappingProperties {
  *
  * @public
  */
-export type SavedObjectsFieldMapping = EsMappingProperty & {
-  /**
-   * The dynamic property of the mapping, either `false` or `'strict'`. If
-   * unspecified `dynamic: 'strict'` will be inherited from the top-level
-   * index mappings.
-   *
-   * Note: To limit the number of mapping fields Saved Object types should
-   * *never* use `dynamic: true`.
-   */
-  dynamic?: false | 'strict';
-  /**
-   * Some mapping types do not accept the `properties` attributes. Explicitly adding it as optional to our type
-   * to avoid type failures on all code using accessing them via `SavedObjectsFieldMapping.properties`.
-   */
-  properties?: Record<EsPropertyName, EsMappingProperty>;
-};
+export type SavedObjectsFieldMapping = EsMappingProperty &
+  EsMappingPropertyBase & {
```

Comment:
The new `MappingSemanticTextProperty` doesn't extend base, so it is missing the optional `fields` field